src/test/java/server/RouterTest.java

```diff
@@ -42,6 +44,22 @@ public void canProcessGETRequestIfRouteHasBeenAdded()  {
         assertEquals("HTTP/1.1 200 OK\r\n", getResponse.getStatusLine());
     }
 
+    @Test
+    public void returns404IfRouteHasNotBeenAdded()  {
+        Request notFoundRequest = new Request(new StatusLine(GET, "/not_found", VERSION));
+        Response getResponse = router.generateResponse(notFoundRequest);
+
+        assertEquals("HTTP/1.1 404 Not Found\r\n", getResponse.getStatusLine());
+    }
+
+    @Test
+    public void canProcessProxyRequestIfRouteHasBeenAdded()  {
+        Request proxyRequest = new Request(new StatusLine(GET, "/http://www.ctabustracker.com/bustime/api/v2/getpredictions", VERSION));
+        Response getResponse = router.generateResponse(proxyRequest);
+
+        assertEquals("HTTP/1.1 200 OK\r\n", getResponse.getStatusLine());
```

Comment:
What's the object under test for this namespace? Wondering if it's potentially redundant to test a variety of paths if you're stubbing the handler - not that that's a bad thing, but perhaps signals that we could use some other tests to exercise these changes

Reply:
I meant to update line 57 to have the query parameters in the test request path - I was aiming to test that it would find the match against the defined endpoint in the `router` when the request path has dynamic query parameters.